app/src/apis/memberApi.ts

```diff
@@ -24,19 +24,22 @@ export class FetchMemberAPI implements MemberAPI {
     }
     const { show } = useToastStore()
 
-    newMembers.map((member: Partial<Member>) => {
+    for (const member of newMembers) {
       if (!isAddress(member.walletAddress)) {
         useErrorHandler().handleError(new Error(`Invalid wallet address`))
```

Comment:
I suggest that you just throw the error and handle them where you're using or calling the classes, it's simpler.